packages/main/src/themes/MultiComboBox.css

```diff
@@ -23,6 +23,10 @@
 }
 
 /* Workaround for IE */
+[ui5-multi-combobox] {
```

Comment:
Does this belong to the current change?

Reply:
No, it does not, it came from the merge, now it won't be displayed after I updated the branch.
 However, it was the right selector, but now everything is fine